.coveragerc

```diff
@@ -0,0 +1,5 @@
+[paths]
+source = 
+	uaclient/
+	ubuntu-pro-client/uaclient/
+	/usr/lib/python3/dist-packages/uaclient/
```

Comment:
I understand `uaclient` and `/usr/lib/...` but why do we need `ubuntu-pro-client/uaclient` ?

Reply:
It was added for testing relational paths, removed now